packages/app-desktop/plugins/GotoAnything.tsx

```diff
@@ -236,35 +233,20 @@ class Dialog extends React.PureComponent<Props, State> {
 
 	public componentWillUnmount() {
 		if (this.listUpdateIID_) shim.clearTimeout(this.listUpdateIID_);
-		document.removeEventListener('keydown', this.onKeyDown);
 
 		this.props.dispatch({
 			type: 'VISIBLE_DIALOGS_REMOVE',
 			name: 'gotoAnything',
 		});
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	public onKeyDown(event: any) {
-		if (event.keyCode === 27) { // ESCAPE
-			this.props.dispatch({
-				pluginName: PLUGIN_NAME,
-				type: 'PLUGINLEGACY_DIALOG_SET',
-				open: false,
-			});
-		}
-	}
-
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	private modalLayer_onClick(event: any) {
-		if (event.currentTarget === event.target) {
-			this.props.dispatch({
-				pluginName: PLUGIN_NAME,
-				type: 'PLUGINLEGACY_DIALOG_SET',
-				open: false,
-			});
-		}
-	}
+	private modalLayer_onDismiss = () => {
```

Comment:
Dismissing the dialog with <kbd>escape</kbd> or clicking outside of the dialog is now handled by `Dialog.tsx`. As such, this logic has been removed from `GotoAnything.tsx`.